demo/mobile/README.md

```diff
@@ -1,16 +1,55 @@
-# FeedHenry RainCatcher Mobile [![Build Status](https://travis-ci.org/feedhenry-raincatcher/raincatcher-demo-mobile.png)](https://travis-ci.org/feedhenry-raincatcher/raincatcher-demo-mobile)
+# FeedHenry RainCatcher Mobile
 
 This is a reference/demo implementation of the Mobile Client application of a WFM project.
 
-## Setup (locally)
+## Local Setup
 
 `npm install`
 
-## Starting (locally)
+## Running on a browser
+
+Being a cordova-based application, it is possible to test this application on a local browser:
+
+`npm start`
+
+or
 
 `grunt serve:local --url=http://localhost:8001`
 
-## Emulator
+The `url` parameter should be the root of the API server application running the [`fh-sync`](https://github.com/feedhenry/fh-sync) endpoints required by the running mobile demo application. See the (demo-server)[https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/demo/server/] for an implementation of the server that is expected to be available.
+
+## Running on a device emulator
+
+To run this application on a device emulator, setup the appropriate native SDKs and run the app through the following commands:
 
 `cordova prepare`
 `cordova emulate android`
+
+See the [Apache Cordova project documentation](https://cordova.apache.org/docs/) for details on how to configure your environment for running applications on a device emulator.
+
+## Repository structure
+
+### `src/app`
+Main folder containing the logic for the application. The app is mainly concerned in providing the setup and configuration values for the RainCatcher and angularjs modules that are used by it as well as for the [FeedHenry JavaScript SDK](https://github.com/feedhenry/fh-js-sdk).
```

Comment:
We using js-sdk just to get the actual cloud URL.  [FeedHenry JavaScript SDK]
Best not to mention it.